src/index.js

```diff
@@ -1,15 +1 @@
-import _ from 'lodash';
-import './style.css';
-
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, currently included via a script, is required for this line to work
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+import './style.css';
```

Comment:
- [ ] Use `index.js` to create all the elements and also remember to use `arrow functions` instead of the `function` keyword. As `ES6` brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more and one of the feature of `ES6` is `arrow function`.

Reply:
@Dipeshtwis Thank you for your review, good call out on the vscode folder. The thing is I do not think we are looking at the same project requirements. There is no requirement that states that I have to load my page dynamically at this moment. The requirement states as follows "Write plain HTML markup with minimum styling (just to make the layout) because you will implement styles in the following steps. " The above-outdated code should not be used as a source of feedback, as you can see it was deleted. Your feedback is poorly explained in regards to the linters and you seem to have some requirements that I don't. Kindly enlighten me.

Reply:
As I have already shared the link of [javascript linters](https://github.com/microverseinc/linters-config/tree/master/javascript), which shows in blue color, if you are not clear why you have to use [javascript linters](https://github.com/microverseinc/linters-config/tree/master/javascript) and not [HTML+CSS+JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js), Let me explain a bit more. As you know you are doing `javascript` project, in this module most of the work is happening in `index.js`, and `HTML` is kind of empty, which will make errors in lighthouse and webhint, so it's better to remove them.

Reply:
@Dipeshtwis you are confusing me. Thanks anyway.

Reply:
Hi @the-catalystmc,

I just saw your latest commit and I feel good that what you have done is absolutely right. :+1:

Reply:
@Dipeshtwis. Yeah, explanations were lost in translation but I got some clarity.